### PR TITLE
Add doc about relationships

### PR DESCRIPTION
**Pull request by Altreus**
This should be enough to address #163

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/FCO/Red.svg?branch=master)](https://travis-ci.org/FCO/Red)
+[![Build Status](https://travis-ci.org/Altreus/Red.svg?branch=master)](https://travis-ci.org/Altreus/Red)
```
Comment (Altai-man) [inline README.md]: This is probably a thinko.

Comment (Altreus) [reply README.md]: Ah, that'll be `mi6` being clever

Comment (Altreus) [reply README.md]: Should I just remove the README commit and only PR the POD?

Comment (Altai-man) [reply README.md]: That's for @FCO to decide, I just pointed the thing out so it won't be accidentally merged. :+1:

Comment (FCO) [reply README.md]: Thank you for noticing! :)

**Comment by FCO** [comment]
Thank you very much!

**Pull request merged**